### DISCUSSION TITLE
File location on windows 10

### DISCUSSION THREAD

**Discussion by General8** [debauchee/barrier#1465] [Q&A]
I uninstalled barrier, and reinstalled it and still have my same configuration. I made sure the barrier folder in program files was deleted and also checked AppData/local. I deleted all temp files and whenever I reinstall, its still there.

Am I missing a file somewhere? On linux I know there is a file in config but unsure about Windows.

**Comment by Taknok** [comment]
To fully clean barrier you have to:
1. Uninstall barrier
2. Remove the `Barrier` folder in `AppData/local`
3. Remove the `barrier.[random]` file in `AppData/Temp`
4. Remove the keys in the registry with `regedit.exe`. The keys are under `Computer\HKEY_USERS\S-[long Windows build number of current version]\SOFTWARE\Debauchee`
5. Remove the firewall rules (if full remove is wanted)

**Reply by llinfeng** [reply]
The Step 4 is very helpful - on both my Win 10 machines, the key is named as: `Computer\HKEY_CURRENT_USER\SOFTWARE\Debauchee`.

Upon removing this entry, reinstalling Barrier is like a breeze. (Version 2.4.0 worked fine, for example, when both the server and the client are opting not to use SSL certificates.)

**Reply by HyeAnn-Lee** [reply]
In my case, there were nothing in `AppData/Temp` but Step 1, 3, 4 worked for me.
In my Win 11 machine, the key is named as : `Computer\HKEY_CURRENT_USER\Software\Debauchee`.

**Reply by nbolton** [reply]
Barrier is no longer in development. Check out Deskflow (upstream) or Input Leap (fork).

https://github.com/deskflow/deskflow
https://github.com/input-leap/input-leap

If this is still an issue in those projects, we would appreciate a cross-post of this issue.
